src/test/resources/test_data/responses/eligibility-and-suitability-view.json

```diff
@@ -11,6 +11,8 @@
     "responsibleCom": null,
     "team": null,
     "policyVersion": "1.0",
+    "optOutReasonType":null,
```

Comment:
Will add decent values here, just want you guys to have a look, to tell me if in general I'm on the right track for this change?